### ISSUE TITLE
Hi,do you have a ideal that support the plugin to search from the jcenter and make it more useful for AndroidStudio?

### ISSUE THREAD

**Issue by itxuye** [siosio/GradleDependenciesHelperPlugin#1]
Hi,do you have a ideal that support the plugin to search from the jcenter and make it more useful for AndroidStudio?

**Comment by Drjacky** [comment]
@itxuye I agree with you. I've found [another](https://github.com/bestwu/gradle-dependencies-plugins-helper-plugin) plugin which uses JCenter.

If I found a free time, I'll fork this plugin from @siosio, add this feature and send a pull request.

I'll appreciate you if you could help me on this. :pray:
